src/cs50/sql.py

```diff
@@ -32,12 +33,23 @@ def __init__(self, url, **kwargs):
             if not os.path.isfile(matches.group(1)):
                 raise RuntimeError("not a file: {}".format(matches.group(1)))
 
-        # Create engine, raising exception if back end's module not installed
-        self.engine = sqlalchemy.create_engine(url, **kwargs)
+            foreign_keys_enabled = kwargs.pop("foreign_keys_enabled", False)
+
+            # Create engine, raising exception if back end's module not installed
+            self.engine = sqlalchemy.create_engine(url, **kwargs)
+
+            # Whether to enable foreign key constraints
```

Comment:
For consistency, I would change this to an imperative: "Enable foreign key constraints". (The conditional aspect can be inferred from the `if`.)